app/models/ems_cluster.rb

```diff
@@ -336,4 +336,8 @@ def self.non_openstack_clusters_exists?
   def openstack_cluster?
     ext_management_system.class == ManageIQ::Providers::Openstack::InfraManager
   end
+
+  def self.display_name(number = 1)
+    n_('Cluster / Deployment Role', 'Cluster / Deployment Roles', number)
```

Comment:
Should the plural be "Clusters / Deployment Roles"?

Reply:
Shouldn't "Cluster" also be pluralized?

Reply:
Yes, this has been corrected to `'Clusters / Deployment Roles'`.